apps/condo/domains/meter/schema/MeterReportingPeriod.js

```diff
@@ -66,46 +66,37 @@ const MeterReportingPeriod = new GQLListSchema('MeterReportingPeriod', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
-        start: {
+        startAt: {
             schemaDoc: 'Starting day of the month for sending a push about the need to submit meter readings',
             type: Integer,
             defaultValue: 20,
             isRequired: true,
             hooks: {
                 validateInput: async ({ context, operation, resolvedData }) => {
                     if (operation === 'create' || operation === 'update') {
-                        if (resolvedData.start > 31 || resolvedData.start < 1) {
+                        if (resolvedData.startAt > 31 || resolvedData.startAt < 1) {
                             throw new GQLError(ERRORS.INVALID_START, context)
                         }
                     }
                 },
             },
         },
 
-        finish: {
+        finishAt: {
```

Comment:
We're using `At` postfix to specify, that field data is date time in ISO format. This field breaks this consistency, which is not good for third-party developers.

If you want to specify, that it is day of month, not specific date, include that in fieldName...Something like `finishDay` will be ok